packages/webtest/webtest.1.0.1/descr

```diff
@@ -0,0 +1 @@
+Framework for in-browser testing of js_of_ocaml code
```

Comment:
this looks like a really cool library! A bit more description here about how it works might make it more accessible to casual users.